From ea7c1ab6ea538c7e4f5a6ea6b0a6d2d1db13cc33 Mon Sep 17 00:00:00 2001
From: Richard Fitzgerald <rf@opensource.wolfsonmicro.com>
Date: Mon, 1 Sep 2014 14:07:13 +0100
Subject: [PATCH 313/351] ASoC: wm8998: Enable DVFS for sample rate 2 and 3
 controls

Higher sample rates on domains 2 and 3 must enable
DVFS boost

Change-Id: I8c610cb0b18791ca93f4865918b50bcc341ec59b
Signed-off-by: Richard Fitzgerald <rf@opensource.wolfsonmicro.com>
---
 sound/soc/codecs/wm8998.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/codecs/wm8998.c b/sound/soc/codecs/wm8998.c
index 4de0cc9..2df44af 100644
--- a/sound/soc/codecs/wm8998.c
+++ b/sound/soc/codecs/wm8998.c
@@ -201,8 +201,8 @@ SOC_ENUM("LHPF2 Mode", arizona_lhpf2_mode),
 SOC_ENUM("LHPF3 Mode", arizona_lhpf3_mode),
 SOC_ENUM("LHPF4 Mode", arizona_lhpf4_mode),
 
-SOC_VALUE_ENUM("Sample Rate 2", arizona_sample_rate[0]),
-SOC_VALUE_ENUM("Sample Rate 3", arizona_sample_rate[1]),
+ARIZONA_SAMPLE_RATE_CONTROL_DVFS("Sample Rate 2", 2),
+ARIZONA_SAMPLE_RATE_CONTROL_DVFS("Sample Rate 3", 3),
 
 SOC_VALUE_ENUM("ISRC1 FSL", arizona_isrc_fsl[0]),
 SOC_VALUE_ENUM("ISRC2 FSL", arizona_isrc_fsl[1]),
-- 
1.7.9.5

